src/tests/multihost/ad/test_adparameters.py

```diff
@@ -692,8 +692,10 @@ def test_0017_gssspnego_adjoin(self, multihost):
         multihost.client[0].run_command(remove_pcap)
         assert status == 'PASS'
 
+    @staticmethod
     @pytest.mark.tier1
-    def test_0018_bz1734040(self, multihost, adjoin):
+    @pytest.mark.skip(reason="Does not work on RHEL 9, skip until resolved.")
```

Comment:
This will skip to run for OS that is not rhel9. We'll have to use @pytest.mark.skipif() I think

Reply:
Well, not sure how we skip based on the rhel/fedora version. But if this MR is kept open until mass merge to rhel-8.5 branch is done it should work.

Reply:
I added a check on client[0] distro instead.